extensions/amp-accordion/1.0/amp-accordion.css

```diff
@@ -29,3 +29,12 @@
   padding-right: 20px;
   border: solid 1px #dfdfdf;
 }
+
+amp-accordion.i-amphtml-display-locking > section:not([expanded]) > :last-child,
+amp-accordion.i-amphtml-display-locking
+  > section:not([expanded])
+  > :last-child
+  * {
+  content-visibility: hidden-matchable !important;
+  display: block !important;
+}
```

Comment:
The CSS for supporting display locking in AMP mode is separate from Preact CSS (handled by JSS file).  This follows our current pattern of handling AMP CSS separately allowing for the publisher to easily overwrite to customize.  Although in this case, we probably do not want to have them overwrite, I think it still makes sense to follow the existing pattern here.

Reply:
Another point of duplication here. Ok for now, but let's leave a comment explaining the duplication so that we can revisit it later.

Reply:
Added a comment in the latest revision.